homeassistant/components/intent/__init__.py

```diff
@@ -74,6 +86,13 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
         NevermindIntentHandler(),
     )
     intent.async_register(hass, SetPositionIntentHandler())
+    intent.async_register(hass, SetTimerIntentHandler())
```

Comment:
Create Timer?

Reply:
Renamed to "start" to fit with the terminology elsewhere.